resources/views/components/profile/flair.blade.php

```diff
@@ -2,7 +2,7 @@
     <span
         aria-label="User role"
         class="text-xs border px-3 py-0.5 rounded-full mr-1.5 cursor-default"
-        title="This user is {{ Str::title($user->flair->value) }}"
+        title="{{ Str::title($user->flair->getDescription()) }}"
```

Comment:
What is the purpose of calling `Str::title()` here? It was there because the `flair->value` was lowercase but since you replaced it with description, you need to remove `Str::title()` from here.